src/test/resources/features/login.feature

```diff
@@ -0,0 +1,61 @@
+ #language: ru, en
+Feature: Sign in
+
+  Background:
+    Given Main page is opened
+
+  @Smoke
+  Scenario: TC-1, Test to check if the entrance form opens
+    When Open the entrance form
+    Then Check if the entrance form was opened: "через социальные сети" appears
+
+  @Ignore
```

Comment:
Why is this tag not specified in the runner by default? I think tests with this tag should be ignored by default

Reply:
fixed